client/ayon_resolve/api/plugin.py

```diff
@@ -746,3 +745,13 @@ def _store_new_instance(self, new_instance):
 
         # Add instance to current context
         self._add_instance_to_context(new_instance)
+
+
+def get_representation_files(representation):
```

Comment:
Where is this used?

Reply:
https://github.com/ynput/ayon-resolve/blob/feature/AY-979_finish_Resolve_new_publisher/client/ayon_resolve/plugins/load/load_clip.py#L43-L44

Reply:
Yep this is not new code, I realized this was missing from parent PR when testing the load features.